app/jobs/animals_job.rb

```diff
@@ -0,0 +1,40 @@
+class AnimalsJob < ApplicationJob
```

Comment:
Still going to used `delayed_job` to parse this all, output can be stored in new table.